### ISSUE TITLE
Feature Request: Acquiring Class Probabilities/Probability Masks

### ISSUE THREAD

**Issue by jacksteussie** [open-mmlab/mmsegmentation#3629]
Not supported yet.

But we would add metric which may use logits (such as AUC) this quarter.

_Originally posted by @MengzhangLI in https://github.com/open-mmlab/mmsegmentation/issues/1278#issuecomment-1033686123_
            

Was wondering if this was still a planned feature?
